docs/old/user-guide.md

```diff
@@ -0,0 +1,53 @@
+# Ed-Fi-X-ODS-Performance
```

Comment:
This is a copy of an old file, text is unchanged. I completely rewrite the user guide in the parent `docs` directory.